cli/src/cli/__init__.py

```
@@ -0,0 +1,43 @@
+"""Temporal Pipeline CLI."""
+
+from importlib import metadata
+
+import typer
+from rich import print as rprint
+
+try:
+    from cli.command import pipeline_app, plugin_app, worker_app
+except ImportError:
+    raise typer.Exit(1)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)